crates/relayer/src/chain/endpoint.rs

```diff
@@ -596,6 +596,33 @@ pub trait ChainEndpoint: Sized {
                     IncludeProof::Yes,
                 )?;
 
+                (maybe_packet_proof, channel_proof)
+            }
+            PacketMsgType::TimeoutOnCloseOrdered => {
+                let channel_proof = {
+                    let (_, maybe_channel_proof) = self.query_channel(
+                        QueryChannelRequest {
+                            port_id: port_id.clone(),
+                            channel_id: channel_id.clone(),
+                            height: QueryHeight::Specific(height),
+                        },
+                        IncludeProof::Yes,
+                    )?;
+                    let channel_merkle_proof = maybe_channel_proof.expect(QUERY_PROOF_EXPECT_MSG);
```

Comment:
This is consistent with the proof handling throughout the file.

Reply:
Yes, I see that this is done in a few places in this function. My question was whether it would be worthwhile to change these `expect` calls to follow our regular error handling convention. I can make those changes to this branch if you'd prefer.

Reply:
Ah i see. imo we should open another issue/ PR and understand why it was done this way (and approved as is) initially and why it is wrong.

Reply:
> we should open another issue/ PR and understand why it was done this way
Opened #3082 to capture this issue 👍